README.md

```diff
@@ -35,6 +36,24 @@ module.exports = fp(function (fastify, opts, next) {
 
 You can check [here](https://github.com/npm/node-semver#ranges) how to define a `semver` range.
 
+You can also pass some metadata thatr will be handled by Fastify, such as the dependencies of your plugin.
+```js
+const fp = require('fastify-plugin')
+
+function plugin (fastify, opts, next) {
+  // your plugin code
+  next()
+}
+
+module.exports = fp(plugin, {
+  version: '0.x',
+  dependencies: {
+    fastify: ['plugin1', 'plugin2'],
+    reply: ['compress']
+  }
```

Comment:
I'm a bit lost on how this dependencies block would be used.

Reply:
Is still a WIP, checkout https://github.com/fastify/fastify/pull/466.